### ISSUE TITLE
Convoy not starting, nothing in the logs

### ISSUE THREAD

**Issue by amalagaura** [rancher/convoy#151]
I might be having devicemapper issues, but I am getting this error when trying to start convoy.

I am just giving a summary because I am unable to copy paste right now:

```
driver: devicemapper, msg: "", reason: "prepare"
msg: Found created pool, skip pool reinit
msg: Cleaning up environment, pkg: daemon
msg: EOF
```

I have dedicated block devices for metadata and data.

Any clue what I can do to resolve this with devicemapper? Is there any way to check or rebuild the metadata ?

Running version 0.4.3

**Comment by amalagaura** [comment]
I think the data volumes might be corrupted. Is there any way to test them? Should I be able to mount the separate metadata and data volumes? I tried but that is failing. Are they ext4 ?

**Comment by yasker** [comment]
data volume and metadata volume are not ext4 fs, so you won't able to mount them. 

You can try
```
dmsetup remove convoy-pool
```

Then try again.

Also, includes the full log may help.

**Comment by amalagaura** [comment]
Thanks for the comment. I did try removing the convoy-pool device but it appears there was some low level corruption in this pool along with a thin pool i was using for docker. In fact the docker thin pool corruption was so bad it caused a kernel panic when starting docker. I wiped the convoy pool and just started from scratch.

One comment is that LVM thin pools are the way docker recommends now. They no longer recommend devicemapper thin pool in their documentation. They appear to be better supported than devicemapper thin pools (they appear to be different somehow). There are a few tools for LVM thin pool repair but there are no such tools from devicemapper thin pool repair.

Thanks for the product!
